packages/web-react/src/components/Toast/README.md

```diff
@@ -315,44 +331,48 @@ This hook returns:
 
 | Name       | Type                                                         | Default    | Description                                         |
 | ---------- | ------------------------------------------------------------ | ---------- | --------------------------------------------------- |
+| `clear`    | `() => void`                                                 | () => {}   | Function that will clear toast queue                |
 | `color`    | [[Emotion Color dictionary][dictionary-color] \| `inverted`] | `inverted` | Color variant                                       |
-| `hide`     | `function`                                                   | () => {}   | Function that will hide UncontrolledToast           |
+| `hide`     | `(toastId) => void`                                          | () => {}   | Function that will hide UncontrolledToast           |
 | `iconName` | `string`                                                     | —          | Name of a custom icon to be shown along the message |
 | `id`       | `string`                                                     | `''`       | ToastBar ID                                         |
 | `isOpen`   | `bool`                                                       | `false`    | Open state of UncontrolledToast                     |
 | `message`  | [`string` \| `ReactNode`]                                    | null       | Message inside UncontrolledToast                    |
-| `show`     | `function`                                                   | () => {}   | Function that will show UncontrolledToast           |
+| `show`     | `(message, toastId, options?) => void`                       | () => {}   | Function that will show UncontrolledToast           |
 
-#### How to use `showToast` function:
+#### How to use `show` function:
 
 This function has two required parameters: message and ID.
 All other options are not required and can be omitted entirely.
 
 ```jsx
+const { show } = useToast();
+
          ┌─⫸ Message inside UncontrolledToast (required)
          │
          │                 ┌─⫸ ToastBar ID (required)
          │                 │
 show('Toast message', 'toast-id', {
-  color: 'danger',         // Color variant, default: 'inverted'
+  color: 'danger',         // Color variant, default: 'inverted' \*
```

Comment:
Earlier the `\*` used to be next to the hasIcon, is this the correct place?

Reply:
i think it is

Reply:
it is not, i'll place it to the hasIcon row